tidb-lightning.toml

```diff
@@ -104,7 +104,10 @@ checksum-table-concurrency = 16
 [post-restore]
 # if set true, checksum will do ADMIN CHECKSUM TABLE <table> for each table.
 checksum = true
-# if set true, compact will do compaction to tikv data.
+# if set to true, compact will do level 1 compaction to tikv data.
+# if this setting is missing, the default value is also true.
+level-1-compact = true
```

Comment:
So if tikv/tikv#4199 is merged, the sample TOML file should provide `false` as the recommended value.